doc/version-history.rst

```diff
@@ -13,6 +13,7 @@ Requires:
 * ts_salobj 7
 * ts_idl 3.2
 * IDL file for DIMM from ts_xml 11
+* Add stubs for new commands
```

Comment:
v6.0.0 already has been tagged. I'd prefer this to be v6.0.1 instead.

Reply:
ok